src/span-data.ts

```diff
@@ -114,6 +117,23 @@ export abstract class BaseSpanData implements Span {
  */
 export class RootSpanData extends BaseSpanData implements types.RootSpan {
   readonly type = SpanType.ROOT;
+  // Locally-tracked list of children.
+  private children: ChildSpanData[] = [];
+  // A function that replaces the ChildSpanData#endSpan prototype method on
+  // a child span instance.
+  private static overrideChildEndSpanHandler = function(
+      this: ChildSpanData, timestamp?: Date) {
```

Comment:
Can you use an identifier other than `this`?

Reply:
`this` is a type annotation only, it'll be erased on compilation.